react/Autosuggest/Autosuggest.js

```diff
@@ -122,10 +124,10 @@ export default class Autosuggest extends Component {
     };
 
     const allInputProps = {
+      ...omit(this.props, [ 'inputProps', 'labelProps', 'autosuggestProps' ]),
       ref: this.storeTextFieldReference,
       inputProps: enrichedInputProps,
-      labelProps: enrichedlabelProps,
-      ...omit(this.props, [ 'inputProps', 'autosuggestProps' ])
```

Comment:
Spreading first and last could yield different output yes. Im not expecting there to be any overriding by design, but wanted to know if the reason for moving the spread was to fix a problem caused by overriding.